ledger/eval.go

```diff
@@ -411,6 +412,8 @@ func (eval *BlockEvaluator) transactionGroup(txgroup []transactions.SignedTxnWit
 			txWithoutGroup.ResetCaches()
 
 			group.TxGroupHashes = append(group.TxGroupHashes, crypto.HashObj(txWithoutGroup))
+		} else if len(txgroup) > 1 {
+			return fmt.Errorf("transactionGroup: [%d] had zero Group but was submitted in a group of %d", gi, len(txgroup))
```

Comment:
I think `goal clerk rawsend` might run into trouble here: right now it uploads the entire file as one group to the REST API, regardless of whether it has group fields set or not.

Reply:
what's the right level for separating behavior between an atomic group of txns vs a bunch of unrelated txns? I think Adam raised the issue that we shouldn't have the situation where a proposer might treat a bunch of txns that arrived together as an atomic set but a verifier only pieces back together the atomicity group by the Group field. Though, I guess at that point the whole group is in the block, presumably correctly, and other txns around it are also in the block and it doesn't matter why.

Reply:
Actually, I looked more closely at `goal clerk rawsend`, and it looks like it already does the right thing.  So, never mind!  I think it'll all just work with your change.